stage1/init/init.go

```diff
@@ -165,6 +165,10 @@ func installAssets() error {
 	if err != nil {
 		return err
 	}
+	systemdSysusersBin, err := common.LookupPath("systemd-sysusers", os.Getenv("PATH"))
```

Comment:
I don't know if we should use PATH, but if we don't, `systemctl` and `bash` should be changed as well for consistency.

I'd say LGTM since it is fixing the issue and using $PATH is not introduced by this patch.